drafts/use-cases/1.entry-point.md

```diff
@@ -28,18 +28,27 @@ HTTP 200 OK
 ```
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@id": "/api",
     "@type": "hydra:EntryPoint",
-    "events": {
-        "@id": "/api/events", 
+    "#events": {
+        "@id": "/api/events",
         "title": "List of events",
         "@type": "hydra:Collection",
-        "operation": {
-            "@type": "hydra:Operation",
-            "title": "Create new event",
-            "method": "POST",
-            "expects": "schema:Event"
-        }
+        "operation": [
+            {
+               "@type": ["hydra:Operation", "schema:ViewAction"],
+                "title": "List events",
+                "method": "GET",
```

Comment:
I'm with @tpluscode here. The "open world assumption" of RDF is a bit too much to take in and won't go well with most developers. On one side we have undefined operations that may or may not work as discussed here, on the other we have terms in referenced vocabularies that may or may not be present.

Being able to scope the implementation of a client application to what the server **actually has to offer** is imho very important. Saying that the scope is infinite is of course very flexible, but it makes the task of implementing a sensible client application well nigh impossible.